### DISCUSSION TITLE
Support for Markdown JavaScript (mdjs) Format

### DISCUSSION THREAD

**Discussion by mmc41** [11ty/eleventy#4078] [Enhancement Queue]
**Is your feature request related to a problem? Please describe.**
I would like support for [Markdown JavaScript (mdjs)](https://open-wc.org/mdjs/)

**Comment by Ryuno-Ki** [comment]
Given #1103 this could be turned down (as a head's up)

**Comment by zachleat** [comment]
`open-wc` is using Eleventy 😅.

But yes we can put this into the enhancement queue. This will almost certainly live in plugin-land though. (see also #117)

**Comment by zachleat** [comment]
> This repository is now using [lodash style issue management](https://twitter.com/samselikoff/status/991395669016436736) for enhancements. This means enhancement issues will now be closed instead of leaving them open.

> View [the enhancement backlog here](https://github.com/11ty/eleventy/issues?utf8=%E2%9C%93&q=label%3Aneeds-votes+sort%3Areactions-%2B1-desc+). Don’t forget to upvote the top comment with 👍!

**Comment by zachleat** [comment]
I believe the updated link for this one is here: https://rocket.modern-web.dev/tools/markdown-javascript/overview/
